mysql-test/t/audit_log_filter_users_PS-5707.test

```diff
@@ -0,0 +1,63 @@
+# Windows does not have grep
```

Comment:
Not required with search_pattern.

Instead:
* `not_embedded`
* and the `wait_until_count_sessions` pattern (see other tests)8

Reply:
Done